docs/running-on-kubernetes.md

```diff
@@ -1203,17 +1203,17 @@ See the [configuration page](configuration.html) for information on Spark config
   <td>3.0.0</td>
 </tr>
 <tr>
-  <td><code>memoryOverheadFactor</code></td>
+  <td><code>spark.kubernetes.memoryOverheadFactor</code></td>
   <td><code>0.1</code></td>
   <td>
-    This sets the Memory Overhead Factor that will allocate memory to non-JVM memory, which includes off-heap memory allocations, non-JVM tasks, various systems processes, and <code>tmpfs</code>-based local directories when <code>local.dirs.tmpfs</code> is <code>true</code>. For JVM-based jobs this value will default to 0.10 and 0.40 for non-JVM jobs.
+    This sets the Memory Overhead Factor that will allocate memory to non-JVM memory, which includes off-heap memory allocations, non-JVM tasks, various systems processes, and <code>tmpfs</code>-based local directories when <code>spark.kubernetes.local.dirs.tmpfs</code> is <code>true</code>. For JVM-based jobs this value will default to 0.10 and 0.40 for non-JVM jobs.
     This is done as non-JVM tasks need more non-JVM heap space and such tasks commonly fail with "Memory Overhead Exceeded" errors. This preempts this error with a higher default.
     This will be overridden by the value set by <code>spark.driver.memoryOverheadFactor</code> and <code>spark.executor.memoryOverheadFactor</code> explicitly.
```

Comment:
should spark.executor.memoryOverheadFactor this be spark.kubernetes.executor.memoryOverheadFactor ?

Reply:
This is `Spark Standalone` documetation, @bjornjorgensen ~ 😄

Reply:
It seems that you are looking at the first commit. I removed K8s part from this PR completely at the latest commit.

Reply:
Here.
```
$ git diff HEAD~2 --stat
 docs/spark-standalone.md | 10 ++++++----
 1 file changed, 6 insertions(+), 4 deletions(-)
```

Reply:
yes, I did read the K8s part.